docs/src/stats.json

```diff
@@ -0,0 +1,84 @@
+{
+  "0.2.6": {
+    "components": 15,
+    "subcomponents": 7,
+    "Accordion": {
+      "max": "13.2 kB",
+      "min": "6.94 kB",
+      "gzip": "2.81 kB"
+    },
```

Comment:
Here's the output we're getting (see the bottom for library level max/min/gzip size).